CMakeLists.txt

```diff
@@ -130,6 +131,25 @@ include_directories(${CMAKE_CURRENT_BINARY_DIR})
 link_directories(${LLVM_BUILD_LIBRARY_DIR})
 add_definitions(${LLVM_DEFINITIONS})
 
+
+#-------------------------------------------------------------------------------
+# Sanitizier configuration
+#-------------------------------------------------------------------------------
+
+if(STABLEHLO_ENABLE_SANITIZER)
+  string(TOLOWER "${STABLEHLO_ENABLE_SANITIZER}" STABLEHLO_ENABLE_SANITIZER_LOWERCASE)
+  if (STABLEHLO_ENABLE_SANITIZER_LOWERCASE STREQUAL "address")
+    add_compile_options(-fsanitize=address -fsanitize=undefined -fsanitize=leak -fno-omit-frame-pointer)
```

Comment:
To my untrained eye, this looks like it's turning on more than just the `address` sanitizer. Is this intended?

Reply:
Yes -- they seem to be a common coupling from what I found online researching best practice.
The grouping otherwise is when sanitizers themselves can't run together in same session.
Memory & Address for instance are such a duo -- same for Thread.